### PR TITLE
Implemented GET request for TELEMETRY

### LINKED ISSUE #176: MCP - GET for Telemetry
References #175 

Given: TSS is running.
When: MCP performs a GET request to `/json_data/teams/0/TELEMETRY.json`.
Then: MCP caches the JSON response.

### PR DESCRIPTION
**Pull request by dignojrteogalbo**
# Description

Uses the requests library to perform a GET request to `/json_data/teams/0/TELEMETRY.json`.

Fixes #176 

## Bug Fix or Feature

- [ ] Bug
- [X] Feature
- [ ] Other

# Testing Instructions

- [ ] Navigate to `ARSIS-Telemetry-GroundControl/groundcontrol-api/`
- [ ] Run the test suite using `docker-compose up --build tests`
- [ ] The test for `test_get_telemetry` should pass.

## Linting

- [ ] I've linted my code

### CODE REVIEW

File: ARSIS-Telemetry-GroundControl/groundcontrol-api/app/tss.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
-endpoint = "localhost:14141"
+import requests
+
+endpoint = "http://localhost:14141"
```
Comment (Aetheris743) [inline ARSIS-Telemetry-GroundControl/groundcontrol-api/app/tss.py:3]:
```suggestion
from os import getenv
endpoint = getenv("TSS_ENDPOINT")
```

**Review by Aetheris743 — CHANGES_REQUESTED** [review]
Do we want to use environment variables at this stage in development? Eventually, it will be useful for deployment and testing

**Review by Aetheris743 — CHANGES_REQUESTED** [review]
This will make changing environments easier

**Comment by dignojrteogalbo** [comment]
> This will make changing environments easier

I've added your suggested change and included an environment variable in the `docker-compose.yml` for the tests.

**Review by Aetheris743 — APPROVED** [review]
Looks good to me

**Pull request merged**